src/main/pages/che-7/installation-guide/con_target-platforms.adoc

```diff
@@ -1,17 +1,12 @@
 [id="target-platforms_{context}"]
```

Comment:
If it's not included anywhere, should we get rid of it?

Reply:
This will be deleted.

Reply:
@MichalMaler, please, before deleting existing content, always track down its origin (who merged and why, associated issue, etc.). It may be that the content is needed, but it wasn't properly incorporated into the docs structure.

Reply:
@rkratky there's no content, it's an empty file. It was either a placeholder for something or an artifact. Both should not be in the master ideally.

Reply:
This was an empty template. I am checking it with doc plans. If it isn't there and @themr0c doesn't know about, I am deleting it. In a case it's actual content, I do the same, buck backing up the content and asking you or Radim for further action.